composer.json

```diff
@@ -4,33 +4,51 @@
     "description": "Easycredit module",
     "license": "MIT",
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.4",
+        "spryker-shop/checkout-page": "^2.2.0 || ^3.21.0",
+        "spryker-shop/shop-application": "^1.13.0",
+        "spryker/calculation": "^4.12.0",
         "spryker/checkout-extension": "^1.0.0",
         "spryker/guzzle": "^2.2.0",
         "spryker/kernel": "^3.31.0",
+        "spryker/money": "^2.10.0",
+        "spryker/quote": "^2.16.0",
         "spryker/sales": "^8.17.0 || ^10.0.0 || ^11.0.0",
         "spryker/step-engine": "^3.2.0",
         "spryker/symfony": "^3.2.2",
+        "spryker/transfer": "^3.32.0",
```

Comment:
Please use minimal required version with needed functionality. For `spryker/transfer` it should be `^3.25.0`
Please also check all other dependencies